lalrpop/src/build/mod.rs

```diff
@@ -365,7 +365,19 @@ fn emit_recursive_ascent(
         exit(1);
     }
 
+    // Find a better visibility for some generated items.
+    // We try to find a good enough common denominator for all starting nonterminals.
```

Comment:
It seems what you do is to compute the max of the visibility of all non terminals. In this case, I would just spell this out simply, instead of "try to find a good enough common denominator", which is IMHO a bit vague.

Reply:
You're right, I changed the comment 🙂